Tools/AP_Periph/can.cpp

```diff
@@ -102,6 +108,11 @@ static struct instance_t {
 #elif CONFIG_HAL_BOARD == HAL_BOARD_SITL
     HALSITL::CANIface* iface;
 #endif
+
+#ifdef HAL_PERIPH_CAN_MIRROR
+    ObjectBuffer<AP_HAL::CANFrame> *mirror_queue;
+    uint8_t mirror_fail_count;
```

Comment:
is uint8 enough here?

Reply:
If we hit 8 fails we drop the packet, same as what's done for the normal transmit process.